index.html

```diff
@@ -809,6 +809,21 @@ <h2 id="collections">Collection Functions (Arrays or Objects)</h2>
 =&gt; [[1, 3, 5], [0, 2, 4]]
 </pre>
 
+      <p id="isEqual">
+        <b class="header">isEqual</b><code>_.isEqual(list, other)</code>
+        <br />
+        Performs an optimized deep comparison between one <b>list</b> and the <b>other</b>, to determine
```

Comment:
Actually in this case I think object methods or utility is more correct as
it has multiple behaviors based on type switching
On Jul 31, 2015 4:32 PM, "Gábor Luk" <notifications@github.com> wrote:

> In index.html
> <https://github.com/jashkenas/underscore/pull/2255#discussion_r36013522>:
>
> > @@ -809,6 +809,21 @@ <h2 id="collections">Collection Functions (Arrays or Objects)</h2>
> >  =&gt; [[1, 3, 5], [0, 2, 4]]
> >  </pre>
> >
> > +      <p id="isEqual">
> > +        <b class="header">isEqual</b><code>_.isEqual(list, other)</code>
> > +        <br />
> > +        Performs an optimized deep comparison between one <b>list</b> and the <b>other</b>, to determine
>
> The documentation consistently uses *list* for every function in the
> *Collections* section. It serves as a shorthand for "array or object".
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jashkenas/underscore/pull/2255/files#r36013522>.
>